File: find_numbers.js
const fs = require('fs');
const path = require('path');

const dirPath = "C:\\Users\\loq\\.gemini\\antigravity\\scratch\\hotel-advantage-inn";
const patterns = [
  /wa\.me\/\d+/,
  /tel:[\d\-\+]+/,
  /\b\d{10}\b/,
  /\d{5}\s*\d{5}/,
  /\+91/
];

function scanDir(dir) {
  const files = fs.readdirSync(dir);
  for (const file of files) {
    const fullPath = path.join(dir, file);
    const stat = fs.statSync(fullPath);
    if (stat.isDirectory()) {
      scanDir(fullPath);
    } else if (file.endsWith('.html') || file.endsWith('.js') || file.endsWith('.css') || file.endsWith('.json')) {
      try {
        const content = fs.readFileSync(fullPath, 'utf-8');
        const lines = content.split('\n');
        lines.forEach((line, idx) => {
          for (const pattern of patterns) {
            if (pattern.test(line)) {
              console.log(`${file}:${idx + 1}: ${line.trim()}`);
              break;
            }
          }
        });
      } catch (err) {}
    }
  }
}

scanDir(dirPath);

File: find_numbers.py
import os
import re

dir_path = r"C:\Users\loq\.gemini\antigravity\scratch\hotel-advantage-inn"
patterns = [
    re.compile(r"wa\.me/\d+"),
    re.compile(r"tel:[\d\-\+]+"),
    re.compile(r"\b\d{10}\b"),
    re.compile(r"\d{5}\s*\d{5}"),
    re.compile(r"\+91")
]

for root, dirs, files in os.walk(dir_path):
    # Skip images or binary assets
    for file in files:
        if file.endswith(('.html', '.js', '.css', '.txt', '.json')):
            file_path = os.path.join(root, file)
            try:
                with open(file_path, 'r', encoding='utf-8') as f:
                    lines = f.readlines()
                    for idx, line in enumerate(lines):
                        for p in patterns:
                            if p.search(line):
                                print(f"{file}:{idx+1}: {line.strip()}")
                                break
            except Exception as e:
                pass
